ctwrap/_version.py

```diff
@@ -1,9 +1,10 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
 """Versioning information/credits
 
 https://setuptools.readthedocs.io/en/latest/setuptools.html#specifying-your-project-s-version
 """
 
-__author__ = "I. Schoegl"
 __version_info__ = (0, 1, 1)
```

Comment:
this should probably be changed to
```
__version_info__ = (0, 2, 0, 'b')
```
with `b` for *beta* ... once this is merged and the documentation is updated, we can drop the 'b' again.